AGENTS.md

```
@@ -0,0 +1,80 @@
+```
+
+## Dependencies
+
+```bash
+pip install -r requirements.txt
+# aiohttp, cloudscraper, Jinja2, pyrofork, pymongo, psutil, python-dotenv, speedtest-cli, tgcrypto, uvloop==0.21.0
+```
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)